packages/fxa-auth-server/lib/routes/subscriptions/stripe-webhook.ts

```diff
@@ -472,7 +472,10 @@ export class StripeWebhookHandler extends StripeHandler {
       return;
     }
     const account = await Account.findByUid(uid, { include: ['emails'] });
-    if (!account) {
+    // If the request has a request id, it means that our API triggered this so
+    // we can safely ignore the account not existing as this is typically due to
+    // us deleting the account in FxA.
+    if (!account && !event.request.id) {
```

Comment:
I can't see how we can avoid throwing sentry bugs perpetually in that case. Because to truly know if it was the case of 5442, we'd have to check if the Stripe user exists in say, 5 minutes when the delete is done. We have no system to push work into the future like that.

My thought for 5442 was that we would add a hook on the invoice.paid to see if the user no longer existed in our system, and alert then, rather than here, in customer.updated. I also can't see how/why we'd want to report an error if we're intentionally manipulating Stripe customer records that we don't have Firefox Accounts for.